src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier supplier = new ColorSupplier();
+        ball.setColor(supplier.getRandomColor());
+        ball.setNumber(new Random().nextInt(100));
```

Comment:
let's make global Random and ColorSupplier fields and use them here

Reply:
Why do we need to create global Random and ColorSupplier fields?